MLP_Model/MLP_utils/config.toml

```diff
@@ -1,13 +1,15 @@
 [DEFAULT]
```

Comment:
Consider using a more specific table name here based on what's appropriate to the goals for this toml file.

Reply:
changed to a more specific name